### PR TITLE
Correct installation of requirements for updating available software

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

File: .github/workflows/update_available_software.yml [hunk 1]
```diff
@@ -24,13 +24,13 @@ jobs:
...
           . venv/bin/activate
-          pip install -r requirements.txt
+          pip install -r scripts/available_software/requirements.txt
 
           # copy current JSON data, so we can compare after updating it
           cp docs/available_software/data/json_data.json docs/available_software/data/json_data.json.orig
           cp docs/available_software/data/json_data_detail.json docs/available_software/data/json_data_detail.json.orig
 
-          python scripts/available_software/available_software.py
+          python scripts/available_software/requirements.txt
```
Comment (trz42) [inline .github/workflows/update_available_software.yml:33]: Is this change intended?

**Review by trz42 — CHANGES_REQUESTED** [review]
Running
```
python scripts/available_software/requirements.txt
```
is maybe not intended?

File: .github/workflows/update_available_software.yml (hunk 1 above)
Comment (ocaisa) [reply .github/workflows/update_available_software.yml:33]: Nope! Whoops, wrong spot

**Comment by ocaisa** [comment]
Hmm, closing this, but my impression is it is installing the wrong `requirements.txt`, but I can't see where

**Comment by ocaisa** [comment]
Ah, I had one thing right and one wrong!

File: .github/workflows/update_available_software.yml (hunk 1 above)
Comment (ocaisa) [reply .github/workflows/update_available_software.yml]: Fixed!

**Review by trz42 — APPROVED** [review]

**Comment by boegel** [comment]
Ah, overlooked this in #195, thanks!
